rails-commenting.rb

```diff
@@ -6,25 +6,25 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) BlogPostController is a class which inherits from ApplicationController and has methods. It usesd for responding the request and returning the output
```

Comment:
You appear to have a good handle on the controller methods and the action performed. As well as what the instance variables are doing in each example.